gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,8 @@
+#Thu Jan 09 16:51:17 TRT 2025
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionSha256Sum=d725d707bfabd4dfdc958c624003b3c80accc03f7037b5122c4b1d0ef15cecab
```

Comment:
FYI if you change distributonUrl, you also have to update distributionSha256Sum. I will fix this